### ISSUE TITLE
Contributors: consent to relicensing for moving gps into dep

### ISSUE THREAD

**Issue by sdboyer** [sdboyer/gps#215]
Dearest lovely contributor peoples,

In order to move gps directly into dep and hack on it - which was something that was always kinda the goal anyway - we need to relicense the code; golang/dep#300. The Google lawyers have indicated it should be sufficient to have people reply to this thread with "I agree to relicense my contributions", and then make sure you've [signed the Google CLA](https://cla.developers.google.com/).

- [ ] @spenczar 
- [ ] @ReSTARTR 
- [ ] @narqo 
- [ ] @freeformz 
- [ ] @erizocosmico 
- [ ] @jstemmer 
- [ ] @zbintliff 
- [ ] @kris-nova 
- [ ] @tomwilkie 
- [ ] @chx 
- [ ] @zellyn 
- [ ] @adg 
- [ ] @kamilchm 
- [ ] @sakeven 
- [ ] @carolynvs (as I'll likely merge #213 before the move happens)

**Comment by erizocosmico** [comment]
I agree to relicense my contributions
On Thu, 13 Apr 2017 at 19:06, sam boyer <notifications@github.com<mailto:notifications@github.com>> wrote:

Dearest lovely contributor peoples,

In order to move gps directly into dep and hack on it - which was something that was always kinda the goal anyway - we need to relicense the code; golang/dep#300<https://github.com/golang/dep/issues/300>. The Google lawyers have indicated it should be sufficient to have people reply to this thread with "I agree to relicense my contributions", and then make sure you've signed the Google CLA<https://cla.developers.google.com/>.

  *    @spenczar<https://github.com/spenczar>
  *    @ReSTARTR<https://github.com/ReSTARTR>
  *    @narqo<https://github.com/narqo>
  *    @freeformz<https://github.com/freeformz>
  *    @erizocosmico<https://github.com/erizocosmico>
  *    @jstemmer<https://github.com/jstemmer>
  *    @zbintliff<https://github.com/zbintliff>
  *    @kris-nova<https://github.com/kris-nova>
  *    @tomwilkie<https://github.com/tomwilkie>
  *    @chx<https://github.com/chx>
  *    @zellyn<https://github.com/zellyn>
  *    @adg<https://github.com/adg>
  *    @kamilchm<https://github.com/kamilchm>
  *    @sakeven<https://github.com/sakeven>
  *    @carolynvs<https://github.com/carolynvs> (as I'll likely merge #213<https://github.com/sdboyer/gps/pull/213> before the move happens)

**Comment by zellyn** [comment]
"I agree to relicense my contributions".

I thought I'd signed the CLA already, but it let me do it again, so I did.

**Comment by zbintliff** [comment]
I agree to relicense my contributions

**Comment by carolynvs** [comment]
I agree to relicense my contributions

**Comment by spenczar** [comment]
I agree to relicense my contributions

**Comment by narqo** [comment]
I agree to relicense my contributions.

**Comment by kamilchm** [comment]
I agree to relicense my contributions

**Comment by tomwilkie** [comment]
I agree to relicense my contributions

**Comment by chx** [comment]
I agree to relicense my contributions

**Comment by sdboyer** [comment]
Look at these rapid responses - awesome, thank you!

Could folks also please confirm that they've signed Google's CLA?

**Comment by chx** [comment]
I signed the Google CLA (is this issue the gps equivalent of drupalconfessions :P ? )

**Comment by freeformz** [comment]
I agree to relicense my contributions and I've previously signed the Google CLA.

**Comment by jstemmer** [comment]
I agree to relicense my contributions and they're already covered by the Google CLA.

**Comment by ReSTARTR** [comment]
I agree to relicense my contributions and I've signed the Google CLA.

**Comment by sakeven** [comment]
I agree to relicense my contributions

**Comment by sdboyer** [comment]
@sakeven and you've signed the Google CLA?

**Comment by kris-nova** [comment]
I agree to relicense my contributions and I've signed the Google CLA - how exciting!

**Comment by sakeven** [comment]
@sdboyer yes

**Comment by sdboyer** [comment]
@erizocosmico you've signed the Google CLA, right?

**Comment by sdboyer** [comment]
consent from @adg - https://twitter.com/enneff/status/853363967384707072

**Comment by erizocosmico** [comment]
Yep, I did back when I did my first PR on dep.
On Sat, 15 Apr 2017 at 23:46, sam boyer <notifications@github.com<mailto:notifications@github.com>> wrote:

@erizocosmico<https://github.com/erizocosmico> you've signed the Google CLA, right?

**Comment by sdboyer** [comment]
@erizocosmico ah right, i 'd forgotten you'd done one there, too. OK, looks like we're all set...

**Comment by fabulous-gopher** [comment]
This issue was moved to golang/dep#413
